public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -45,6 +50,8 @@ export default class FindingDetailsFlyout extends Component<
     super(props);
     this.state = {
       loading: false,
+      ruleViewerFlyoutShown: false,
```

Comment:
This boolean seems redundant since we are anyways rendering the flyout based on the data. I think just `ruleViewerFlyoutData` is enough for check. Thoughts?

Reply:
Totally right. I've cleaned it up